### PR TITLE
Restructure docs, stage 1

### LINKED ISSUE #1467: Describe Tarantool features in common DBMS terms
Tarantool is a complex system that stores user data.
I think users want to know guarantees that Tarantool provides before start to use it. Unfortunately we don't have any description of tarantool guarantees in common DBMS terms, like 'consistency model', 'transation isolation level', 'ACID/BASE', 'MVCC' etc.

I've tried to search this in documentation and failed.

What isolation levels tarantool uses? Search in documentation found only two relevant pages: [Transaction control](https://www.tarantool.io/en/doc/2.5/book/box/atomic/) and [SQL](https://www.tarantool.io/en/doc/2.5/book/sql/). Both pages doesn't contain answers for the question. On page about [SQL](https://www.tarantool.io/en/doc/2.5/book/sql/): "SET TRANSACTION statement ISOLATION SERIALIZABLE clause | SET TRANSACTION ISOLATION LEVEL SERIALIZABLE; | Fail. Syntax error.". Well, but what isolation levels are supported then?

[Transaction control](https://www.tarantool.io/en/doc/2.5/book/box/atomic/) describes atomicity support and isolation. What about durability and consistency?

This [page](http://jepsen.io/consistency) contains good description of possible consistency models.

### LINKED ISSUE #2487: [0pt] Add supported architecture description
# Feature name

Supported architecture

**Product:** Tarantool
**Since:** all
**Audience/target:** Developers, DevOps
**Root document:** whatever you want
**SME:** @tsafin 
**Peer reviewer:** I don't know

# Details

*   Write about officially supported hardware architectures by Tarantool. If there are several types of support (like official/only-CI/only several manual tests) - describe all of them. Just tell somehow to the end-users, on which target architectures they run Tarantool
*   Related issue: https://github.com/tarantool/tarantool/issues/5471

# Definition of done

- [ ] Add documentation about the supported hardware architectures to the documentation

# Planning checklist

- [ ] Pick product label:
  - [server](https://github.com/tarantool/doc/labels/server) for Tarantool core

- [ ] Pick type label: 
  - [reference](https://github.com/tarantool/doc/labels/reference),
  - [getting started](https://github.com/tarantool/doc/labels/getting%20started),
  - [user_guide](https://github.com/tarantool/doc/labels/user_guide), etc. 
- [ ] Estimate complexity in storypoints in the title
- [ ] Add to **Documentation** board → **Backlog**

Feel free to adjust the issue, as you want

CC @tsafin

### PR DESCRIPTION
**Pull request by patiencedaur**
Part of this epic: https://github.com/tarantool/doc/issues/2051
Part of https://github.com/tarantool/doc/issues/2805
Resolves https://github.com/tarantool/doc/issues/2487
Resolves https://github.com/tarantool/doc/issues/1467

Deployment: https://docs.d.tarantool.io/ru/doc/restructure-docs-1/

- Releases
- [x] Rename Release notes to Releases
- [x] Add a link to Downloads to the Releases page
- [x] Structure release notes by series (2.10, 1.10, EOL versions)
- [x] Replace links accordingly
- [x] Remove RN of outdated prerelease versions

- Overview
- [x] Write the overview article "What is Tarantool?"
- [x] Add the Overview section

- How-to guides
- [x] Add the How-to guides section
- [x] Move "What's next?" tutorials from Getting started to the new section
- [x] Remove the old Tutorials section and put all the tutorials into How-to guides
- [x] Add more tutorials as per https://www.notion.so/tarantool/Tarantool-CE-c075d0db9ce24893beca704c145ae501#704bee0eff07402fa4db192ef76923d8

See https://www.notion.so/tarantool/Tarantool-CE-c075d0db9ce24893beca704c145ae501#fe407fc7c3184d50b0bef804c760b719.

### CODE REVIEW

File: locale/ru/LC_MESSAGES/reference/reference_lua/box_index/pairs.po [hunk 1]
```diff
@@ -34,14 +34,15 @@ msgstr ""
 "все кортежи в порядке по возрастанию или по убыванию, начиная с указанного "
 "ключа. Однако другие типы индексов не поддерживают упорядочивание."
 
+#, fuzzy
```
Comment (NickVolynkin) [inline locale/ru/LC_MESSAGES/reference/reference_lua/box_index/pairs.po]: This update breaks translations in a lot of places. For example, page https://docs.d.tarantool.io/ru/doc/restructure-docs-1/reference/reference_lua/box_index/pairs/

Comment (patiencedaur) [reply locale/ru/LC_MESSAGES/reference/reference_lua/box_index/pairs.po]: Looks like make update-po worked in an unexpected way. I can just remove this commit. In this case, what is the correct way to batch update translations?

**Comment by patiencedaur** [comment]
Redirect after this PR:

- https://www.tarantool.io/en/doc/latest/tutorials/ -> https://www.tarantool.io/en/doc/latest/howto/
- https://www.tarantool.io/ru/doc/latest/tutorials/ -> https://www.tarantool.io/ru/doc/latest/howto/

.

- https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/examples/ -> https://www.tarantool.io/en/doc/latest/how-to/crud
- https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_space/examples/ -> https://www.tarantool.io/ru/doc/latest/how-to/crud

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_db/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_db/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_db/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_db/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_connectors/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_connectors/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_connectors/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_connectors/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_python/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_python/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_python/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_php/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_php/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_php/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_php/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_go/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_go/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_go/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_go/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_cxx/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_cxx/
- https://www.tarantool.io/ru/doc/latest/getting_started/getting_started_cxx/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_cxx/

.

- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_cartridge/ -> https://www.tarantool.io/en/doc/latest/how-to/getting_started_cartridge/
- https://www.tarantool.io/en/doc/latest/getting_started/getting_started_cartridge/ -> https://www.tarantool.io/ru/doc/latest/how-to/getting_started_cartridge/

.

- https://www.tarantool.io/en/doc/latest/reference/reference_sql/sql_beginners_guide/ ->  https://www.tarantool.io/en/doc/latest/how-to/sql/sql_beginners_guide
- https://www.tarantool.io/ru/doc/latest/reference/reference_sql/sql_beginners_guide/ -> https://www.tarantool.io/ru/doc/latest/how-to/sql/sql_beginners_guide

.

- https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/examples/ -> https://www.tarantool.io/en/doc/latest/how-to/crud
- https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_space/examples/ -> https://www.tarantool.io/ru/doc/latest/how-to/crud

**Comment by p7nov** [comment]
More redirects:

* https://www.tarantool.io/en/doc/latest/release/2.10.0-rc1/ > https://www.tarantool.io/en/doc/latest/release/2.10.0/
* https://www.tarantool.io/en/doc/latest/release/2.10.0-beta1/ > https://www.tarantool.io/en/doc/latest/release/2.10.0/
* https://www.tarantool.io/en/doc/latest/release/2.10.0-beta2/ > https://www.tarantool.io/en/doc/latest/release/2.10.0/

**Comment by patiencedaur** [comment]
How to make redirects (internal document): https://www.notion.so/tarantool/5b0e1ab45d0841a3b502cfea89829ef7

**Pull request merged**